cmd/demo/node.go

```diff
@@ -115,7 +113,11 @@ func (n *node) peer(addr perunpeer.Address) *peer {
 }
 
 func (n *node) setupChannel(ch *client.Channel) {
-	perunID := ch.Peers()[0]
+	if len(ch.Peers()) > 2 {
```

Comment:
I think this check does not make sense, since *go-perun* can only do two-party channels in *Despina*.  
The version of *go-perun* is fixed anyway, so there is no way this changes unless we update it. If you still want to have the check, you also need to do it in the `HandleProposal` function.

Reply:
i vote for including both checks, when initiating a channel opening and when receiving a proposal. the reason is that a developer who uses our code should be aware that the code only works for the two-party case. this is why i added also comments on the respective lines of code.

Reply:
Well, the code is not meant to be used by other developers since we do not have docs or tests (and it is just a demo). It also does not work for sub-channels, so another test to check `Locked` for zero-sum would be needed or a comment.